pages/cart-page/cart.html

```diff
@@ -0,0 +1,140 @@
+                                    </div>
+                                </div>
+                                <button class="button button-primary">Move to Wishlist</button>
+                                <button class="button button-outline">Remove from Cart</button>
+                            </div>
+                        </div>
+                    </div>
+
+                </div>
+                <div class="cart-transactions">
+                    <div class="title heading-3">Price Details</div>
+                    <hr>
+                    <div class="sub-price">
+                        <div class="price">
+                            <div class="text">
+                              Price (<span class="quantity">2</span> items)
+                            </div>
+                            <div class="value">₹ 998</div>
+                        </div>
+                        <div class="price">
+                            <div class="text">Discount</div>
+                            <div class="value">- ₹ 299</div>
+                        </div>
+                        <div class="price">
+                            <div class="text">Delivery Charges</div>
+                            <div class="value">₹ 49</div>
+                        </div>
+                    </div>
+                    <hr>
+                    <div class="total-price">
+                        <div class="text">Total Amount</div>
+                        <div class="value">₹ 748</div>
+                    </div>
+                    <hr>
+                    <div class="discount-msg">You will save ₹ 299 on this order</div>
+                    <button class="button button-secondary">
+                        Place order
+                    </button>
+                </div>
+            </section>
+        </section>
+    </main>
+
+    <footer>
+        <div class="about">
+            <a href="/index.html
+            " class="brand-tag">JacketEX</a>
+            <div class="text">Stylish and comfortable Jackets for all occasions</div>
+            <div class="socials">
+                <a href="https://github.com/ganesh-kanchi"><img src="../../assets/github-brands.svg" alt="github logo"></a>
+                <a href="https://twitter.com/Ganesha_K_Dev"><img src="../../assets/twitter-brands.svg" alt="twitter logo"></a>
+                <a href="https://www.linkedin.com/in/ganesh-kanchi-b9bb7a207/"><img src="../../assets/linkedin-brands.svg" alt="linkedin logo"></a>
+            </div>
+        </div>
+        <div class="rapid-links">
+            <div class="heading">Quick Links</div>
+            <div class="sub-heading"><a href="../product-listing-page/products.html">Products</a></div>
+            <div class="sub-heading"><a href="../wishlist-page/wishlist.html">Wishlist</a></div>
+            <div class="sub-heading"><a href="./cart.html">Cart</a></div>
+        </div>
+
+        <div class="contact">
```

Comment:
this generic you can use footer

Reply:
I have used footer once to wrap all elements in footer.